CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # GoodData Ruby SDK Changelog
 
+
+## 0.6.20
+- iterators for retrieval of project, domain, and group objects
+- use query resource inlining for retrieving user filters
+- added support of GoodData PGP SSO
+- project creation token is not taken from the ~/.gooddata file
```

Comment:
Will not be better something like this?

- project creation token is now taken from the ~/.gooddata file